CHANGELOG.md

```diff
@@ -61,7 +61,10 @@ The table below shows which release corresponds to each branch, and what date th
 - [#1261][1261] Misc `run_in_new_terminal` improvements (notably gdb terminated by default)
 - [#1695][1695] Allow using GDB Python API
 - [#1735][1735] Python 3.9 support in safeeval
+- [#1738][1738] Which function support custom search path
+  - process also looks now at `env['PATH']` to find the path for the executable
 
+[1738]: https://github.com/Gallopsled/pwntools/pull/1738
 [1261]: https://github.com/Gallopsled/pwntools/pull/1261
 [1695]: https://github.com/Gallopsled/pwntools/pull/1695
 [1735]: https://github.com/Gallopsled/pwntools/pull/1735
```

Comment:
Sorry for a pedantic reaction, but please keep the links sorted, this helps reviewing changelog updates a lot.
```suggestion
[1261]: https://github.com/Gallopsled/pwntools/pull/1261
[1695]: https://github.com/Gallopsled/pwntools/pull/1695
[1735]: https://github.com/Gallopsled/pwntools/pull/1735
[1738]: https://github.com/Gallopsled/pwntools/pull/1738
```

Reply:
No, thank for the help, being pedantic for good reasons is important.